src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown

```diff
@@ -115,7 +115,9 @@ IMPORTANT: Any administrator can connect to your application from the SAP Concur
 
 #### <a name="landing-page-sign-in-or-up"></a>Sign In or Sign Up
 
-You should use this flow if you have an online account registration form.
+This is not the most common option to use, but this flow can be used if you have an online account registration form and/or you support free trials.
+
+> Note: If you support Free Trials, your Landing Page is required to indicate how long the Trial will last and what the Partner will do with the data collected if the customer declines to move forward as a customer with the Partner. This text is in addition to the other requirements such as checkboxes for the user to select for your Privacy Policy and Terms and Conditions.
```

Comment:
Suggested text: "Note: if you support free trials, your landing page must also indicate the trial duration and how the data collected will be handled should the customer cancel following the free trial period."

The privacy policy and T&Cs should be handled in a separate bullet if we want to call that out.